packages/docs/src/examples/EthBalance.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { formatEther } from '@ethersproject/units'
+import { Mainnet, DAppProvider, useEtherBalance, useEthers, Config } from '@usedapp/core'
+
+const config: Config = {
+    readOnlyChainId: Mainnet.chainId,
+    readOnlyUrls: {
+      [Mainnet.chainId]: 'https://mainnet.infura.io/v3/57fc2c19095745e59ab96a4aa87dada8',
+    },
+}
+
+ReactDOM.render(
+    <DAppProvider config={config}>
+      <EthBalance />
+    </DAppProvider>,
+    document.getElementById('root')
+)
+
+export function EthBalance() {
```

Comment:
We name every example component `App` because every example is a standalone app.
The imports won't collide because you have default exports, so you can still import it as `EthBalance` in MDX.

Reply:
Fine, i revert my changes